tests/Solarium/Tests/QueryType/Extract/RequestBuilderTest.php

```diff
@@ -139,4 +139,26 @@ public function testContentTypeHeader()
         $this->assertEquals($headers,
                             $request->getHeaders());
     }
+
+    public function testDocumentDateTimeField()
+    {
+        $timezone = new \DateTimeZone('Europe/London');
+        $date = new \DateTime('2013-01-15 14:41:58', $timezone);
+        $document = $this->query->createDocument(array('date' => $date));
```

Comment:
Aren't we using the short-form for arrays? (I thought it was covered by the php-cs-styles) :/